src/group/managed_group/mod.rs

```diff
@@ -0,0 +1,276 @@
+use crate::ciphersuite::*;
+use crate::codec::*;
+use crate::creds::*;
+use crate::errors::ConfigError;
+use crate::framing::{sender::*, *};
+use crate::group::*;
+use crate::key_packages::*;
+use crate::messages::{proposals::*, *};
+use crate::tree::node::*;
+
+/// A `ManagedGroup` represents an `MLSGroup` with an easier, high-level API designed to be used in production. The API
```

Comment:
We should probably put the other API behind a feature flag later. Maybe mention that here already and make an issue.

Reply:
I was wondering about that. I think we should keep both, because it might well be that you cannot do everything you want with `MlsGroup`, but happy to discuss.

Reply:
I created https://github.com/openmls/openmls/issues/215 for that.